cmd/rook/userfacing/multus/validation/validation.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/cmd/rook/userfacing/client"
+	"github.com/rook/rook/pkg/daemon/multus"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	"github.com/spf13/cobra"
+)
+
+// defaults
+var (
+	DefaultValidationNamespace = "rook-ceph"
+
+	// 1 mon, 3 osds, 2 mgrs, 1 mds, 1 nfs, 1 rgw, 1 rbdmirror, 1 cephfsmirror,
```

Comment:
Shouldn't have two mgrs on the same node in production
```suggestion
	// 1 mon, 3 osds, 1 mgr, 1 mds, 1 nfs, 1 rgw, 1 rbdmirror, 1 cephfsmirror,
```